packages/blockchain-service/src/constants/BSCommonConstants.ts

```diff
@@ -1,3 +1,3 @@
 export class BSCommonConstants {
-  static readonly DORA_URL = 'https://dora.coz.io'
+  static readonly DORA_URL = 'https://api.coz.io'
```

Comment:
What do you think to rename to `COZ_API_URL`? What's your opinion, @raulduartep?

Reply:
Ok I'll leave it